specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/virtualMachineScaleSet.json

```diff
@@ -5423,6 +5423,13 @@
       "description": "Describes a Virtual Machine Scale Set."
     },
     "VirtualMachineScaleSetVMReimageParameters": {
+      "type": "object",
+      "properties": {
+        "forceUpdateOSDiskForEphemeral": {
+          "type": "boolean",
+          "description": "Parameter to force update ephemeral OS disk for a virtual machine scale set VM"
```

Comment:
You know booleans are discouraged because they are hard to extend in the future right?